code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -79,6 +79,17 @@
 				else
 					C.deconstruct()
 					visible_message("<span class='warning'>[src] chews through the [C].</span>")
+	for(var/obj/item/reagent_containers/food/snacks/cheesewedge/cheese in range(1, src))
+		if(prob(10))
+			var/cap = CONFIG_GET(number/ratcap)
+			if(LAZYLEN(SSmobs.cheeserats) > cap)
+				visible_message("<span class='warning'>[src] carefully eats the cheese, hiding from the [cap] mice on the station!</span>")
+				qdel(cheese)
+				return INITIALIZE_HINT_QDEL
```

Comment:
The copy paste is strong with this one. This is returned if you want something to qdel after initializing.